examples/libp2p/main.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+#include <libp2p/multi/multiaddress.hpp>
+#include <libp2p/peer/peer_id.hpp>
+
+using namespace libp2p;
+
+int main()
+{
+    auto addr = Multiaddress::create("/ip4/192.168.0.1/tcp/8080");
+    std::cout << "address: " << addr.value().getStringAddress() << std::endl;
+
+    // 创建一个 Peer ID
```

Comment:
Could you remove the non ASCII character. I don't know what they mean and I want to prevent Unicode character mismatches